source/includes/_methods.md

```diff
@@ -159,6 +159,47 @@ Field Name | Purpose | Field Format
 `intent` | The intention of the CTA you are inserting. | String
 `setupFunction(meta)` | The payload object for the current vehicle. | Object
 
+## API.insertGalleryContent(target, objOrArray)
+
+> Usage
+
+```javascript
+(function(WIAPI) {
+  var API = new WIAPI('test-integration'); // Note: Replace 'test-integration' with your actual integration identifier.
```

Comment:
I noticed that all the sample code has this `test-integration` set as the id.

But, this is not needed anymore for third parties. When we load their script we set the id (which would be the WISE integration id in the new system) once during the js load process. That id gets used when they instantiate the API inside their file.

Reply:
That seems unrelated to these changes, so I'll make a separate PR.